docs/apache-airflow/howto/operator/python.rst

```diff
@@ -22,8 +22,11 @@
 PythonOperator
 ==============
 
-Use the :class:`~airflow.operators.python.PythonOperator` to execute
-Python callables.
+Use the ``@task`` decorator to execute Python callables.
+
+.. warning::
+    Previous versions of Airflow required using :class:`~airflow.operators.python.PythonOperator`
+    to execute Python callables. The ``@task`` decorater is now the recommended method.
```

Comment:
The last version to _require_ using PythonOperator was 1.10.10; I’d probably rewrite this to

> The `@task` decorator is recommended to the more “classic” `PythonOperator` approach to execute Python callables.

Reply:
Thanks. Please see [`fc66a4e`](https://github.com/apache/airflow/pull/25845/commits/fc66a4efb5d418e0aaee583406aa1fe9b3421621)